pkg/security/probe/probe_bpf.go

```diff
@@ -220,31 +214,35 @@ func (p *Probe) SetEventHandler(handler EventHandler) {
 	p.handler = handler
 }
 
-// DispatchEvent sends an event to probe event handler
-func (p *Probe) DispatchEvent(event *Event) {
+// DispatchEvent sends an event to the probe event handler
+func (p *Probe) DispatchEvent(event *Event, size uint64, CPU int, perfMap *manager.PerfMap) {
+	log.Tracef("Dispatching event %+v\n", event)
```

Comment:
Aren't we logging twice ?

Reply:
Yup, moved the `Dispatch event ...` log from `handleEvent` to `DispatchEvent` to be consistent with `DispatchCustomEvent` 👍🏻